src/constants/actionsColumn.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import type {CellContext, ColumnDef, ColumnDefTemplate} from '@tanstack/react-table';
+
+import type {TableActionsSettings} from '../components';
+import {RowActions} from '../components';
+
+export const ACTIONS_COLUMN_ID = '_actions';
+const ACTIONS_COLUMN_SIZE = 44;
+
+export const getActionsCell = <TData extends unknown>({
+    getRowActions,
+    renderRowActions,
+    rowActionsSize,
+}: TableActionsSettings<TData>): ColumnDefTemplate<CellContext<TData, unknown>> => {
+    const ActionsCell = (props: CellContext<TData, unknown>) => {
```

Comment:
cell in tanstack/react-table is: `(props: CellContext) => any`. We can't declare ActionsCell with custom properties like `getRowActions` without wrapping. And getActionsCell is wrap. Think is ok to declare it here because it should be exported to Public API and it related with columns.

Reply:
`ActionsCell ` is now declared inside the function, it's not a good practice, you can either not declare it at all and just return the needed content or declare it somewhere outside